src/shogun/classifier/svm/LibLinear.cpp

```diff
@@ -116,8 +117,9 @@ bool LibLinear::train_machine(std::shared_ptr<Features> data)
 		}
 	}
 	SGVector<float64_t> w;
+	index_t len =num_feat + 1;
```

Comment:
is this necessary? i mean it's only used the one place or?

Reply:
It's used in this one place only, cause this is what SG_ALLOCATED_MALLOC takes, based on what I have read

Reply:
I could change it however, np